messages/role_annoucement.md

```diff
@@ -0,0 +1,15 @@
+Text for announcement to astropy-dev mailing list
+
+The Coco proposes to [select as appropriate]
```

Comment:
p.s. Still not sure about the capitalization though.
```suggestion
The CoCo proposes to [select as appropriate]
```

Reply:
I don't care either way. I think either could be justified.